assets/training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -10,7 +10,7 @@ wget=={{latest-pypi-version}}
 applicationinsights=={{latest-pypi-version}}
 azureml-automl-core=={{latest-pypi-version}}
```

Comment:
Should we also increment env version for publish?

Reply:
Ack